clkmgr/pub/clkmgr/client_state.h

```diff
@@ -57,10 +31,7 @@ class ClientState
   private:
     std::atomic_bool connected = false; /**< Connection status */
     std::atomic_bool subscribed = false; /**< Subscription status */
-    /**
-     * Session ID
-     */
-    sessionId_t sessionId = InvalidSessionId;
+    sessionId_t sessionId = InvalidSessionId; /**< Session ID */
```

Comment:
Curiosity - where is this InvalidSessionId defined ?

Reply:
in pub/clkmgr/ultility.h
static const sessionId_t InvalidSessionId = (sessionId_t)(-1);